test/attachments_test.rb

```diff
@@ -66,8 +66,8 @@ class ActiveStorage::AttachmentsTest < ActiveSupport::TestCase
 
   test "attach new blobs" do
     @user.highlights.attach(
-      { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" }, 
-      { io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg" })
+      { io: StringIO.new("STUFF"), filename: "town.jpg", content_type: "image/jpg" },
+      io: StringIO.new("IT"), filename: "country.jpg", content_type: "image/jpg")
```

Comment:
This one isn't quite right either. I know that rubucop is trying to enforce here, but it doesn't make sense in this instance. Don't think whatever rule is being enforced here is valuable enough to keep.